cmake/modules/ExternalDependenciesVersions.cmake

```diff
@@ -4,7 +4,7 @@
 set(TTG_TRACKED_VG_CMAKE_KIT_TAG d1b34157c349cf0a7c2f149b7704a682d53f6486)  # provides FindOrFetchLinalgPP and "real" FindOrFetchBoost
 set(TTG_TRACKED_CATCH2_VERSION 3.5.0)
 set(TTG_TRACKED_MADNESS_TAG 93a9a5cec2a8fa87fba3afe8056607e6062a9058)
-set(TTG_TRACKED_PARSEC_TAG 58f8f3089ecad2e8ee50e80a9586e05ce8873b1c)
+set(TTG_TRACKED_PARSEC_TAG a9ab33d8287578c68c0349662352f280bc83e2c0)
```

Comment:
Use the 4.0 please

Reply:
Too many things missing that we need in PaRSEC so that would last for 1 PR:

- https://github.com/ICLDisco/parsec/pull/710
- https://github.com/ICLDisco/parsec/pull/706
- https://github.com/ICLDisco/parsec/pull/695
- https://github.com/ICLDisco/parsec/pull/687 (or related)
- https://github.com/ICLDisco/parsec/pull/681
- https://github.com/ICLDisco/parsec/pull/671

Maybe 4.1 will work for us.